apps/condo/domains/ticket/access/TicketContact.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generated by `createschema ticket.TicketContact 'property:Relationship:Property:PROTECT; unitName:Text; email:Text; phone:Text; name:Text;' --force`
+ */
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+
+async function canReadTicketContacts ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        property: { organization: { employees_some: { user: { id: user.id }, isBlocked: false } } },
+    }
+}
+
+async function canManageTicketContacts ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return false
```

Comment:
Could you please try to use the context param? and stop using direct access to low-level DB API

Reply:
Used in `11428e1e`.